### ISSUE TITLE
Weird errors

### ISSUE THREAD

**Issue by epicgamermanlol** [danielkrupinski/Osiris#11]
Hello, i was wondering if anyone could help me with two errors that i cant seem to fix.

```
C2039	'str': is not a member of 'std::basic_ostream<char,std::char_traits<char>>' (compiling source file Interfaces.cpp)	Osiris	\osiris-master\osiris\interfaces.h	38
```	
and

`C2228	left of '.c_str' must have class/struct/union (compiling source file Interfaces.cpp)	Osiris	\osiris-master\osiris\interfaces.h	38	
`

**Comment by danielkrupinski** [comment]
Hello, it's probably a Visual Studio bug, cause I don't have any problems compling the code. You should consider updating your Visual Studio 2017 installation to the latest version 15.9.4.

**Comment by epicgamermanlol** [comment]
Yeah i tried this last night and just checked again. It worked. Thanks haha
